.github/workflows/ci.yml

```diff
@@ -11,5 +11,9 @@ jobs:
             - uses: actions/checkout@v1
             - name: Compiling
               env:
-                  VFLAGS: -show_c_cmd -cg -statsdho
+                  VFLAGS: -show_c_cmd -cg -stats
```

Comment:
`-stats` is needed/works only for tests, although it will be harmless here, I think it will be cleaner if you remove -stats at this place.